test/integration/scheduler_perf/scheduler_perf_types.go

```diff
@@ -0,0 +1,30 @@
+package benchmark
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+		"k8s.io/kubernetes/pkg/api/v1"
+)
+
+// High Level Configuration for all predicates and priorities.
+type schedulerPerfConfig struct {
+	NodeAffinity nodeAffinity
+	InterpodAffinity interpodAffinity
+}
+
+// interpodAffinity priority configuration details.
+type interpodAffinity struct {
+	Enabled     bool
+	Operator    metav1.LabelSelectorOperator
```

Comment:
I have no idea what the Operator is in the context of Affinity. Is this together with `affinityKey` just a way to implement NodeSelector? Why not include it explicitly? Plus I believe it makes no sense in PodAffinity, as it's used only in the NodeAffinity if I'm not mistaken.